### PR TITLE
fix login issues and add create component

### PR DESCRIPTION
**Pull request by aeddaqqa**
- fix login with mnemoniac phrase issue 
- fix login options lead back to the wallet starting page
- add create wallet process to camino suite application

### CODE REVIEW

File: wallet/src/views/access/Mnemonic.vue [hunk 1]
```diff
@@ -95,10 +96,11 @@ export default class Mnemonic extends Vue {
             this.isLoading = false
             return
         }
-
+        // let parent = this
```
Comment (Ayagoumi) [inline wallet/src/views/access/Mnemonic.vue]: unnecessary comment, please remove it.

**Review by Ayagoumi — CHANGES_REQUESTED** [review]
remove unnecessary comments

**Review by Ayagoumi — APPROVED** [review]

File: wallet/src/views/Create.vue [hunk 2]
```diff
@@ -1,61 +1,65 @@
...
 @use '../styles/main';
 
 .access_view {
-  /*display: grid;*/
-  /*grid-template-columns: 1fr 1fr;*/
-  display: flex;
-  justify-content: center;
-  align-items: center;
+    /*display: grid;*/
+    /*grid-template-columns: 1fr 1fr;*/
```
Comment (Chriserus) [inline wallet/src/views/Create.vue]: Unnecessary comments

File: wallet/src/views/Create.vue [hunk 3]
```diff
@@ -1,61 +1,65 @@
...
+                },
+                {
+                    vmid: 'og:description',
+                    name: 'description',
+                    content: description,
+                },
+                {
+                    vmid: 'og:title',
+                    name: 'og:title',
+                    content: 'Create | Avalanche Wallet',
```
Comment (Chriserus) [inline wallet/src/views/Create.vue]: Please rebrand to Camino Wallet

File: wallet/src/views/Create.vue [hunk 4]
```diff
@@ -1,61 +1,65 @@
...
-      ],
-      title: "Create",
-    };
-  },
+    components: {
+        CreateWallet,
+    },
+    metaInfo: () => {
+        const description =
+            "Here at Avalanche, we've made storing digital assets easy. Create an Avalanche wallet today is see what the buzz is all about!"
```
Comment (Chriserus) [inline wallet/src/views/Create.vue]: Please rebrand to Camino Wallet

**Review by Chriserus — APPROVED** [review]
Small comments, otherwise LGTM

**Review by Chriserus — APPROVED** [review]

**Pull request merged**